be/src/vec/data_types/data_type_time_v2.cpp

```diff
@@ -37,6 +37,7 @@
 #include "vec/runtime/vdatetime_value.h"
 
 namespace doris {
+#include "common/compile_check_begin.h"
```

Comment:
it's cpp why only begin not have end

Reply:
according to https://selectdb.feishu.cn/wiki/FaJgwjAiuiVyKakCRQAcZmONnKe cpp file does not need the end header.